src/style.css

```diff
@@ -1 +1,67 @@
-/* Styles go here */
+:root {
+  --light-blue: #00acdc;
+}
+
+body {
+  margin: 0;
+}
+
+header {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: #fff;
+  padding-right: 50px;
+}
+
+.header__icon a {
```

Comment:
please add classes to all needed elements
![image](https://github.com/mate-academy/layout_moyo-header/assets/22824431/c1234591-626e-4d9c-9038-37f894fff9cf)